doc/api/util.md

```diff
@@ -126,6 +126,10 @@ util.format(1, 2, 3); // '1 2 3'
 
 ## util.inherits(constructor, superConstructor)
 
+_Note: usage of `util.inherits()` is discouraged. Please use ES6 `class` and
+`extends` to get language level inheritance support, including an unbroken
+inheritance chain._
```

Comment:
'discouraged' is good.  I would maybe add that it's incompatible with ES6 class semantics.